marc.py

```diff
@@ -548,6 +552,19 @@ def __init__(self, _db, library, integration):
         self._db = _db
         self.library = library
         self.integration = integration
+        
+    @classmethod
+    def get_storage_settings(cls, _db):
+        integrations = ExternalIntegration.get_integrations_of_goal(
+            _db, ExternalIntegration.STORAGE_GOAL
+        )
+        for integration in integrations:
+            cls.SETTING['options'].append(
+                dict(key=str(integration.id), label=integration.name)
+            )
+        
+        return cls.SETTING
+
```

Comment:
I see, this method gets called at some point and it adds all the existing storage integrations to the list.

This is fine but--again, just thinking out loud as I go through the diff--it seems like you're going to need to have similar code for  setting up the "cover" and "book" ExternalIntegrationLinks for a collection.

This can wait, but the other issue is that not all ExternalIntegrations with STORAGE_GOAL can actually store MARC records. An S3 integration needs to have a MARC bucket defined, and an LCP integration won't be able to store MARC records at all.